cli/test/smokehouse/report-assert.js

```diff
@@ -147,21 +153,25 @@ function findDifferences(path, actual, expected) {
     }
 
     if (key === '_excludes') {
-      // Re-use state from `_includes` check, if there was one.
-      /** @type {any[]} */
-      const arrToCheckAgainst = inclExclCopy || actual;
```

Comment:
IIRC the idea here was to not copy things when _excludes or _includes aren't used. To that end, we could keep this refactor but keep the perf behavior by only setting `inclExclCopy` if `'_excludes` in expected || '_includes` in expected`